plugins/woocommerce/client/admin/client/settings-payments/offline/settings-payments-bacs.tsx

```diff
@@ -0,0 +1,251 @@
+/**
+ * External dependencies
+ */
+import { __ } from '@wordpress/i18n';
+import { useDispatch, useSelect } from '@wordpress/data';
+import {
+	CheckboxControl,
+	TextControl,
+	TextareaControl,
+	CardBody,
+	Button,
+	Card,
+} from '@wordpress/components';
+import { useState, useEffect } from '@wordpress/element';
+import { paymentGatewaysStore, optionsStore } from '@woocommerce/data';
+
+/**
+ * Internal dependencies
+ */
+import '../settings-payments-body.scss';
+import './settings-payments-offline-method.scss';
+import { FieldPlaceholder } from '~/settings-payments/components/field-placeholder';
+import { PaymentSettingsSection } from '~/settings-payments/components/payment-settings-section';
+import { BankAccountsList } from '~/settings-payments/components/bank-accounts-list';
+import { PaymentSettingsLayout } from '~/settings-payments/components/payment-settings-layout';
+import { GatewaySettingsForm } from '~/settings-payments/components/gateway-settings-form';
+import { BankAccount } from '~/settings-payments/components/bank-accounts-list/types';
+
+export const SettingsPaymentsBacs = () => {
```

Comment:
add please a comment about the component

Reply:
maybe call it direct bank transfer so it's more readable?

Reply:
Updated in d0d09a984c. I think I prefer `bacs` because it is the same identifier used elsewhere in the code (more searchable), but I added a comment referring to it as direct bank transfer as well.